app/code/community/Bolt/Boltpay/etc/config.xml

```diff
@@ -198,6 +198,17 @@
         </boltpay>
       </updates>
     </layout>
+    <events>
```

Comment:
This is unnecessary and probably likely very incorrect.  Set any initial status in the payment `initialize` method.  I recommend setting the order state and status to "pending"